apps/studio/src/common/utils.ts

```diff
@@ -178,3 +199,31 @@ export function entityId(schema: string, entity?: TableOrView | Routine) {
   if (entity) return `${entity.entityType}.${schema}.${entity.name}`;
   return `schema.${schema}`;
 }
+
+export function isFile(e: DragEvent) {
+    const dt = e.dataTransfer;
+    for (let i = 0; i < dt.types.length; i++) {
+        if (dt.types[i] === "Files") {
+            return true;
+        }
+    }
+    return false;
+}
+
+export async function getLastExportPath(filename?: string) {
+  const lastExportPath = await SettingsPlugin.get('lastExportPath')
+
+  if (lastExportPath) {
+    return path.join(lastExportPath, filename)
+  }
+
+  if (platformInfo.isMac || platformInfo.isWindows) {
+    return path.join(homedir(), 'Documents', filename)
```

Comment:
I don't think you need to use `app.getPath("documents")` from the electron API as users can move their Documents folder.

Reply:
I actually think it's better to just use `homedir` anyway, as many software engineers don't put sql in the `Documents` folder.